src/Platform/Microsoft.Testing.Platform/Requests/ConsoleTestExecutionRequestFactory.cs

```diff
@@ -3,30 +3,20 @@
 
 using Microsoft.Testing.Platform.CommandLine;
 using Microsoft.Testing.Platform.Helpers;
-using Microsoft.Testing.Platform.Resources;
 using Microsoft.Testing.Platform.TestHost;
 
 namespace Microsoft.Testing.Platform.Requests;
 
-internal sealed class ConsoleTestExecutionRequestFactory(ICommandLineOptions commandLineService, ITestExecutionFilterFactory testExecutionFilterFactory) : ITestExecutionRequestFactory
+internal sealed class ConsoleTestExecutionRequestFactory(ICommandLineOptions commandLineService, ITestExecutionFilter testExecutionFilter) : ITestExecutionRequestFactory
 {
-    private readonly ICommandLineOptions _commandLineService = commandLineService;
-    private readonly ITestExecutionFilterFactory _testExecutionFilterFactory = testExecutionFilterFactory;
-
-    public async Task<TestExecutionRequest> CreateRequestAsync(TestSessionContext session)
+    public Task<TestExecutionRequest> CreateRequestAsync(TestSessionContext session)
     {
-        (bool created, ITestExecutionFilter? testExecutionFilter) = await _testExecutionFilterFactory.TryCreateAsync();
-        if (!created)
-        {
-            throw new InvalidOperationException(PlatformResources.CannotCreateTestExecutionFilterErrorMessage);
```

Comment:
This resource is no longer used. Could you delete it as well?

Reply:
Deleted